src/cmd/vm.rs

```diff
@@ -236,23 +244,24 @@ pub struct ArgsStart {
 }
 
 fn run_start(args: &ArgsStart) -> Result<()> {
-    if args.acloud {
-        run_acloudw(args)?;
+    let port = if args.acloud {
+        run_acloudw(args)?
     } else {
         run_betty_start(args)?;
-        let info = register_dut("127.0.0.1:9222")?;
+        "9222".to_string()
+    };
 
-        println!(
-            "You can connect the VM instance with `lium dut shell --dut {:?}`.",
-            info.id()
-        );
-        println!("To push an Android build a betty VM, run `lium arc flash`.");
-    }
+    let info = register_dut(&format!("127.0.0.1:{port}"))?;
+    println!(
+        "You can connect the VM instance with `lium dut shell --dut {:?}`.",
+        info.id()
+    );
+    println!("To push an Android build a betty VM, run `lium arc flash`.");
 
     Ok(())
 }
 
-fn run_acloudw(args: &ArgsStart) -> Result<()> {
+fn run_acloudw(args: &ArgsStart) -> Result<String> {
```

Comment:
Add a comment to explain what is the return value (and change it to u16)

Reply:
Updated the return value and added a comment!